File: ticketProject5-DistributedC/TicketSystemCC/model/model/Buyer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace model
{
    public class Buyer : Entity<long>
    {
        private long idAR { get; set; }
        private string firstName { get; set; }
        private string lastName { get; set; }
        private long noTickets { get; set; }

        public Buyer(long idAR, string firstName, string lastName, long noTickets)
        {
            this.idAR = idAR;
            this.firstName = firstName;
            this.lastName = lastName;
            this.noTickets = noTickets;
        }

        public long IdAR
        {
            get { return idAR; }
            set { idAR = value; }
        }

        public string FirstName
        {
            get { return firstName; }
            set { firstName = value; }
        }

        public string LastName
        {
            get { return lastName; }
            set { lastName = value; }
        }

        public long NoTickets
        {
            get { return noTickets; }
            set { noTickets = value; }
        }

        public override string ToString()
        {
            return idAR + "," + firstName + "," + lastName;
        }
        public override bool Equals(object o)
        {
            if (this == o) return true;
            Buyer that = (Buyer)o;
            return idAR.Equals(that.idAR) &
                firstName.Equals(that.firstName) &
                lastName.Equals(that.lastName) &
                noTickets.Equals(that.noTickets);
        }

    }
}

File: ticketProject10-WebClient/TS-REST - CLIENT/TicketModel/src/main/java/ticket/model/Buyer.java
package ticket.model;

import org.hibernate.annotations.GenericGenerator;

import javax.persistence.*;
import java.io.Serializable;
import java.util.Objects;

@javax.persistence.Entity
@Table( name = "Buyers" )
public class Buyer extends Entity<Long> implements Serializable {
    private Long idAR;
    private String firstName;
    private String lastName;
    private Long noTickets;

    public Buyer(Long idAR, String firstName, String lastName, Long noTickets) {
        this.idAR=idAR;
        this.firstName = firstName;
        this.lastName = lastName;
        this.noTickets=noTickets;
    }

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", updatable = false, nullable = false)
    @Override
    public Long getId() {
        return super.getId();
    }

    @Override
    public void setId(Long id) {
        super.setId(id);
    }


    @Column(name = "idAR")
    public Long getIdAR(){
        return idAR;
    }

    public void setIdAR(Long idAR){
        this.idAR=idAR;
    }

    @Column(name = "firstName")
    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    @Column(name = "lastName")
    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    @Column(name = "noTickets")
    public Long getNoTickets() {
        return noTickets;
    }

    public void setNoTickets(Long noTichets) {
        this.noTickets = noTichets;
    }

    @Override
    public String toString() {
        return this.getIdAR() + " " + this.getFirstName() + " " + this.getLastName();
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (!(o instanceof Buyer)) return false;
        Buyer that = (Buyer) o;
        return getIdAR().equals(that.getIdAR()) &&
                getFirstName().equals(that.getFirstName()) &&
                getLastName().equals(that.getLastName()) &&
                getNoTickets().equals(that.getNoTickets());
    }

    @Override
    public int hashCode() {
        return Objects.hash(getIdAR(), getFirstName(), getLastName(), getNoTickets());
    }

}
